realsense.tf

```diff
@@ -1,10 +1,12 @@
 locals {
   realsense_team = [
-    "doronhi",
+    "SamerKhshiboun",
+    "IntelRealSense",
+    "Nir-Az"
   ]
   realsense_repositories = [
-    "librealsense-release",
-    "ros2_intel_realsense-release",
+    "librealsense2-release",
+    "realsense-ros-release",
```

Comment:
Are these "renames" of the current release repositories or the addition of two repositories and the removal of two others?

Release repositories cannot be deleted so the old repositories must be archived rather than removed.

Reply:
> Are these "renames" of the current release repositories or the addition of two repositories and the removal of two others?
> 
> Release repositories cannot be deleted so the old repositories must be archived rather than removed.

Renaming of the old release repositories. The new ones links:
https://github.com/IntelRealSense/librealsense2-release
https://github.com/IntelRealSense/realsense-ros-release

Reply:
> Renaming of the old release repositories. The new ones links:

I think that we may be miscommunicating, [IntelRealSense/librealsense2-release](https://github.com/IntelRealSense/librealsense2-release) does not appear to have any common commits from the `master` branch (bloom's central config branch) of [ros2-gbp/librealsense-release](https://github.com/ros2-gbp/librealsense-release) which suggests to me that this is a new release repository and not an extension of [ros2-gbp/librealsense-release](https://github.com/ros2-gbp/librealsense-release).

The same is the case for [ros2-gbp/ros2_intel_realsense-release](https://github.com/ros2-gbp/ros2_intel_realsense-release) and [IntelRealSense/realsense-ros-release](https://github.com/IntelRealSense/realsense-ros-release)

I think the right thing to do is:

1. Archive both
* ros2-gbp/librealsense-release
* ros2-gbp/ros2_intel_realsense-release

2. and then create new ros2-gbp repositories:
* ros2-gbp/librealsense2-release
* ros2-gbp/realsense-ros-release

3. and mirror push the current contents of the IntelRealSense hosted repositories:
* https://github.com/IntelRealSense/librealsense2-release -> https://github.com/ros2-gbp/librealsense2-release
* https://github.com/IntelRealSense/realsense-ros-release -> https://github.com/ros2-gbp/realsense-ros-release

4. finally, once we've verified that you can push releases to the ros2-gbp repositories it's a good idea to archive the ones in IntelRealSense to avoid accidentally pushing to them in the future.


If that plan works for you, let me know and I'll refactor this PR to include the archivals and mirror the current repositories.

Reply:
Hi @nuclearsandwich,

Thanks for the detailed solution you are proposing.
Before answering this question, I need some clarifications from your side:
- How this is going to affect current/old release for the different distros ? (humble, iron, and even EOL distro.. foxy, galactic...)
- What about writ access to gbp repo ? will we have the permissions to write/edit patches and files of our releases ?
- How does this change will affect our future releases ? specially when new ROS distros are out 

Thanks,
Samer

Reply:
> How this is going to affect current/old release for the different distros ? (humble, iron, and even EOL distro.. foxy, galactic...)

Archiving the repositories will make them read-only but keep them available for people using those earlier packages.

> What about writ access to gbp repo ? will we have the permissions to write/edit patches and files of our releases ?

Release repositories are generally _not_ modified by hand, the `bloom` release tool is primarily used to update release repositories based on changes in the upstream source repository.

> How does this change will affect our future releases ? specially when new ROS distros are out

The release process is the same for most ROS 2 packages. Once granted access to the ros2-gbp repositories you'll be able to follow http://docs.ros.org/en/rolling/How-To-Guides/Releasing/Subsequent-Releases.html when making releases.